package.json

```diff
@@ -831,13 +842,13 @@
           },
           "runme.app.baseDomain": {
             "type": "string",
-            "default": "",
+            "default": "platform.stateful.com",
             "scope": "window",
             "markdownDescription": "Base domain to be use for Runme app"
           },
           "runme.app.panel.runme.cloud": {
             "type": "string",
-            "default": "runme.placeholder",
+            "default": "runme.cloud",
```

Comment:
We should give the panel its view and not reuse `runme.cloud`. Let's have `runme.optin` and have it alias `runme.placeholder` for now.